src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/col2im.cpp

```diff
@@ -0,0 +1,271 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "col2im.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/precision_support.h"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+namespace Col2Im {
+using Col2ImSpecificParams =  std::tuple<
+        InputShape,                                         // data shape
+        std::vector<int64_t>,                               // output size values
+        std::vector<int64_t>,                               // kernel size values
+        ov::Strides,                                        // strides
+        ov::Strides,                                        // dilations
+        ov::Shape,                                          // pads_begin
+        ov::Shape                                           // pads_end
+>;
+
+using Col2ImLayerTestParams = std::tuple<
+        Col2ImSpecificParams,
+        ElementType,                                        // data precision
+        ElementType,                                        // index precision
+        ov::test::TargetDevice                              // device name
+>;
+
+using Col2ImLayerCPUTestParamsSet = std::tuple<
+        Col2ImLayerTestParams,
+        CPUSpecificParams>;
```

Comment:
These usings are already defined in the header

Reply:
Removed.